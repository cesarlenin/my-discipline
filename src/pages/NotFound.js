import React, { Component } from 'react'

export class NotFound extends Component {
  render() {
    return (
      <div>
        <p>404 Not Found</p>
      </div>
    )
  }
}

export default NotFound
