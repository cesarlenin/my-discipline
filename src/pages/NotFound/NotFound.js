import React, { Component } from 'react';

export class NotFound extends Component {
  render() {
    return (
      <section>
        <h1>404 Not Found</h1>
      </section>
    )
  }
}

export default NotFound;
