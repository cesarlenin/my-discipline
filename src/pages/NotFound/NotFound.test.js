import React from 'react';
import ReactDOM from 'react-dom';
import NotFound from './NotFound';
import renderer from 'react-test-renderer';

describe('<NotFound />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render( < NotFound /> , div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create( < NotFound /> ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});