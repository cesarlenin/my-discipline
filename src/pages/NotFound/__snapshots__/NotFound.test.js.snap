// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders this UI as expected 1`] = `
<section>
  <h1>
    404 Not Found
  </h1>
</section>
`;
